Cpp/odin-views/app.cpp

```diff
@@ -93,6 +95,91 @@ namespace {
     } c_app_login;
 
 
+    const class app_verify : public fostlib::urlhandler::view {
+    public:
+        app_verify()
+        : view("odin.app.verify") {
+        }
+
+        std::pair<boost::shared_ptr<fostlib::mime>, int> operator () (
+            const fostlib::json &config,
+            const fostlib::string &path,
+            fostlib::http::server::request &req,
+            const fostlib::host &host
+        ) const {
+            const auto paths = fostlib::split(path, "/");
+            if ( paths.size() != 1)
+                throw fostlib::exceptions::not_implemented(__PRETTY_FUNCTION__,
+                    "Must pass app_id in the URL");
+            fostlib::pg::connection cnx{fostgres::connection(config, req)};
+            const auto app_id = paths[0];
+            fostlib::json app = odin::app::get_detail(cnx, app_id);
+            if ( app.isnull() )
+                throw fostlib::exceptions::not_implemented(__PRETTY_FUNCTION__, "App not found");
+
+            if ( req.method() != "POST" )
```

Comment:
Really we should have braces, `{}`, around the body of the `if`. There's a few places here where they're missing.